pipeline/metadata/6.0.yaml

```diff
@@ -284,14 +284,12 @@ suites:
     rhbuild: "6.0"
     inventory:
       openstack: "conf/inventory/rhel-9-latest.yaml"
-      ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
```

Comment:
You can retain this line, it will be useful if in future we decide to move it back to IBM again.

Reply:
ok sure